packages/toolkit/src/createSlice.ts

```diff
@@ -262,6 +262,16 @@ export function createSlice<
   if (!name) {
     throw new Error('`name` is a required option for createSlice')
   }
+
+  if (
+    typeof process !== 'undefined' &&
+    process.env.NODE_ENV === 'development'
+  ) {
+    if(options.initialState === undefined) {
+      throw new Error('initial state must be different of undefined')
```

Comment:
This should probably be a `console.error`, so that (apart from this message) the application behaves 100% the same in dev and prod - there will be a crash down the line either way, but it should probably not a different crash depending on the environment.

Reply:
You're right, let me change this